src/styles/index.css

```diff
@@ -0,0 +1,19 @@
+@import "./card.css";
+@import "./stars.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+
+  --color-white: #fff;
+  --color-white-border: #f3f3f3;
+  --color-secondary: #616070;
+  --color-main-accent: #060b35;
+  --color-blue-accent: #00acdc;
```

Comment:
You can create another css file 'variables.css' and move all variables to it. 
```
variables.css
:root { 
--variable1: ...
--vatiable2: ...
}
```
After that, just import it at the top of style.css

Reply:
I agree with @serentinos, this makes perfect sense to me